scripts/coverage

```diff
@@ -8,4 +8,4 @@ export SOURCE_FILES="broadcaster tests"
 
 set -x
 
-${PREFIX}coverage report --show-missing --skip-covered --fail-under=88
+${PREFIX}coverage report --show-missing --skip-covered --fail-under=78
```

Comment:
Actually, can you move `show-missing`, `skip-covered` and `fail-under` to the `pyproject.toml`?